pkg/search/postgres/query/bool_query.go

```diff
@@ -12,12 +12,15 @@ func newBoolQuery(ctx *queryAndFieldContext) (*QueryEntry, error) {
 	if len(ctx.queryModifiers) > 0 {
 		return nil, fmt.Errorf("modifiers for bool query not allowed: %+v", ctx.queryModifiers)
 	}
-	res, err := parse.FriendlyParseBool(ctx.value)
-	if err != nil {
-		return nil, err
+
+	if !isWildCardOrNullStringQuery(ctx.value) {
```

Comment:
I feel that wildcard search on a boolean field is an odd thing.  What is the use case to search a boolean with a wildcard?  Wildcard simply means don't search it.

Reply:
Although our store implementation defaults to false, postgres does allow bool fields to be null.  Since wildcard translates to a `is not null` query, I think it makes sense to keep it the same.